mlprojects/sklearn-diabetes/train.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license.
+
+import numpy as np
+from sklearn.datasets import load_diabetes
+from sklearn.linear_model import Ridge
+from sklearn.metrics import mean_squared_error
+from sklearn.model_selection import train_test_split
+import mlflow
+import mlflow.sklearn
+
+import matplotlib
+matplotlib.use('Agg')
+import matplotlib.pyplot as plt
+
+alpha = float(sys.argv[1]) if len(sys.argv) > 1 else 0.5
+
+with mlflow.start_run():
```

Comment:
mlflow.start_run() not needed, see other training scripts

Reply:
using sklearn autologging?

Reply:
gotcha, updated it to autologging